doc/usecases/current/copy_paste_within_same_map.md

```diff
@@ -0,0 +1,70 @@
+# Use Case: Copy & Paste of Selection Within the Same Map via CTRL-C & CTRL-V
+
+## Summary
+
+- **Scope:** All Layers, except Base
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can copy and paste a selection of elements, including succeeding crops, within one map.
+- **Status:** Analysis
+- **Assignee:** Christoph N.
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app on a key-controlled device and, on one of his maps, made a selection of elements that he wants to copy and paste into that same map.
+- **Main success scenario:**
+  - The user copies the selection by pressing CTRL-C.  
+    He clicks anywhere on that same map.  
+    He pastes the copied selection into the map by pressing CTRL-V.  
+    The pasted selection of elements is placed in the map at the user's last click position right before pasting.
+- **Alternative scenarios:**
+  - The user pastes a copied selection without having clicked anywhere else on the map after copying.  
+    The pasted selection is placed with a horizontal and vertical offset next to the copied selection.
+  - The user pastes a copied selection twice or more times in a row.  
+    The pasted selections are each placed with a horizontal and vertical offset next to each other.
+  - The user pastes a copied selection while having other element(s) currently selected.  
+    The pasted selection is placed with a horizontal and vertical offset next to the currently selected element(s).
+  - The user pastes a copied selection having the base layer selected and no pasting happens.  
+    He selects another layer and successfully pastes that selection of elements into it.
```

Comment:
1. the sentence should be generalized: base layer is not the only wrong layer
2. Yvonne and I think there should be some information for the user in this case. Let us have a general guidelines that there should be toasts to inform users about that they need to change to layer X so that operation could work. (I know you don't like toasts so much but we think it is okay in these cases).